### DISCUSSION TITLE
Structure for modules scripts

### DISCUSSION THREAD

**Discussion by ebranlard** [OpenFAST/python-toolbox#61] [Polls]
We'll likely add more and more scripts for specific modules (e.g. tools for AeroDyn, ElastoDyn, SubDyn, etc.). 

We are looking for some feedback on where those tools should be placed. Here's some line of thoughts:
- A shorter syntax is likely preferable for the user. 
- Having everything into only one file for each module could be a bit overwhelming. 
- Currently the fast-farm tools are located in pyFAST/fast-farm/*.py
- a structure of the form `pyFAST/modules/elastodyn` mimics the openfast repository, but is longer to type.

**Comment by andrew-platt** [comment]
For option 2, are you thinking of a `glue-codes` as well (for FAST.Farm, OpenFAST, etc)?

**Reply by ebranlard** [reply]
Good point, I've added a note on that. Currently the fast-farm tools are located in pyFAST/fast-farm/*.py

**Comment by ebranlard** [comment]
pyFAST has been renamed to `openfast_toolbox` and migrated here: https://github.com/OpenFAST/openfast_toolbox . 

Please continue the discussion in this new repository. Sorry for the inconvenience.
